src/main/java/se/kth/sda/skeleton/comments/Comment.java

```diff
@@ -21,6 +23,12 @@
     @NotBlank
     private String contentText;
 
+    @ManyToOne
+    @JsonIdentityInfo(generator = ObjectIdGenerators.PropertyGenerator.class, property = "name")
```

Comment:
Does it send only the name or the whole user?

Reply:
It sends the whole user